style.css

```diff
@@ -549,6 +549,12 @@ input::-webkit-input-placeholder {
   padding-left: 16px;
 }
 
+input[type='text'],
+textarea {
+  padding-left: 10px;
+  box-sizing: border-box;
+}
+
 textarea::-webkit-input-placeholder {
   font-weight: 400;
   font-size: 15px;
```

Comment:
[OPTIONAL] I suggest you change the placeholder text color. 

```
input::-webkit-input-placeholder {
  color: #172b4d;
  .....
}
```

to this: `color: #b2bac5;`